thing-url-adapter.js

```diff
@@ -288,11 +288,13 @@ class ThingURLDevice extends Device {
     let auth = '';
     for (const i in JWT_AUTH) {
       if (this.wsUrl.includes(i)) {
-        auth = `?Authorization=Bearer ${JWT_AUTH[i]}`;
+        auth = `?jwt=${JWT_AUTH[i]}`;
         break;
       }
     }
     // -- added '' and {} to get eslint to shut up
+    // this.ws will still be null here if no jwt has been entered
+    // for the current device
     if (this.ws === null) {
```

Comment:
I was mistaken about this earlier. Null check is not needed. 
I had to read the code a couple of times